src/content/chainlink-functions/tutorials/importing-packages.mdx

```diff
@@ -0,0 +1,149 @@
+---
+section: chainlinkFunctions
+date: Last Modified
+title: "Using Imports with Functions"
+metadata:
+  linkToWallet: true
+whatsnext:
+  {
+    "Try out the Chainlink Functions Tutorials": "/chainlink-functions/tutorials",
+    "Read the Architecture to understand how Chainlink Functions operates": "/chainlink-functions/resources/architecture",
+  }
+---
+
+import { Aside, CopyText } from "@components"
+import { Tabs } from "@components/Tabs"
+import ChainlinkFunctions from "@features/chainlink-functions/common/ChainlinkFunctions.astro"
+
+This tutorial demonstrates different ways to import packages and use them to modify data before storing it on-chain. After [OCR](/chainlink-functions/resources/concepts) completes the off-chain processes, it returns the result to your smart contract.
```

Comment:
Nit: Data from a Functions response doesn't necessarily have to be stored on-chain. It could simply be used by the contract and discarded. (For example, maybe if the Functions request checks ownership of an NFT on a different chain, the response value is not stored on-chain, but is used to grant a user ownership of another NFT on the chain where the Functions request occurs)

Reply:
Edited